Examples/ExampleConsole/README.md

```diff
@@ -0,0 +1,9 @@
+# Example<span style="color: #d30971 !important;">Console</span>
```

Comment:
Keep it simple for better maintenance.

```suggestion
# ExampleConsole
```

Reply:
It was to align with [ExampleBlazor/README.md](https://github.com/influxdata/influxdb-client-csharp/blob/1355279a20f6766ade2abbd613f57672d94c439e/Examples/ExampleBlazor/README.md?plain=1#L4) where the title uses the same style. Should I also change it for ExampleBlazor?